docs/en/pact-properties.rst

```diff
@@ -311,38 +311,32 @@ Mass conservation and column deltas
 In some situations, it’s desirable that the total sum of the values in a
 column remains the same before and after a transaction. Or to put it
 another way, that the sum of all updates to a column zeroes-out by the
-end of a transaction. To capture this pattern, we have the
-``conserves-mass`` property which takes a table and column name:
+end of a transaction. To capture this pattern, we can express a “mass
+conservation” property using ``column-delta``:
 
 .. code:: lisp
 
-   (conserves-mass 'accounts 'balance)
+   (= (column-delta accounts 'balance) 0.0)
+
+This property asserts that the “column delta” is zero, where
+``column-delta`` returns a numeric value of the sum of all changes to
+the column during the transaction.
 
 For an example using this property, see “A simple balance transfer
 example” below.
 
-It turns out that ``conserves-mass`` is actually just a trivial
-application of another property called ``column-delta``, which returns a
-numeric value of the sum of all changes to the column during the
-transaction. So ``(conserves-mass 'accounts 'balance)`` is actually just
-the same as:
-
-.. code:: lisp
-
-   (= 0 (column-delta 'accounts 'balance))
-
 We can also use ``column-delta`` to ensure that a column only ever
-increases monotonically:
+increases during a transaction:
```

Comment:
monotonicity is an important property here. Can i suggest we go with "increases monotonically during a transaction"?